_inc/client/general-settings/site-plan.jsx

```diff
@@ -49,10 +50,8 @@ export default connect(
 	( state ) => {
 		return {
 			getSiteConnectionStatus: () => getSiteConnectionStatus( state ),
-			isFetchingSiteData: isFetchingSiteData( state ),
-			getSitePlan: () => {
-				return getSitePlan( state );
-			}
+			isFetchingSiteData: isFetchingSiteData( state.jetpack.siteData ),
+			getSitePlan: () => getSitePlan( state.jetpack.siteData )
```

Comment:
here, too. Path resolution inside the `state` should be accomplished by `isFetchingSiteData` and `getSitePlan`.